integration/test/update/project.yaml

```diff
@@ -0,0 +1,16 @@
+version: 1
+
+test:
+  env:
+    - "CIRCLECI=${CIRCLECI}"
+    - "CIRCLE_SHA1=${CIRCLE_SHA1}"
+    - "CIRCLE_BUILD_NUM=${CIRCLE_BUILD_NUM}"
+    - "CLUSTER_NAME=${CLUSTER_NAME}"
+    - "COMMON_DOMAIN=${COMMON_DOMAIN}"
+    - "GITHUB_BOT_TOKEN=${GITHUB_BOT_TOKEN}"
+    - "IDRSA_PUB=${IDRSA_PUB}"
+    - "KEEP_RESOURCES=${KEEP_RESOURCES}"
+    - "REGISTRY_PULL_SECRET=${REGISTRY_PULL_SECRET}"
+    - "TEST_DIR=${TEST_DIR}"
+    - "TESTED_VERSION=${TESTED_VERSION}"
+    - "VAULT_TOKEN=mytoken"
```

Comment:
What is `mytoken` ? Why do we set it here?

Reply:
Copy/paste. No idea.

Reply:
I think it has to be set to something, but value doesn't matter for dev vault